### DISCUSSION TITLE
Gitpages.app - A tool to help all open source engineers in the world create landing pages

### DISCUSSION THREAD

**Discussion by boyney123** [EddieHubCommunity/support#1692] [Show and tell]
Hi all,

I came over from Discord to share my idea with you all (hopefully thats OK).

My name is Dave Boyne, and I'm working on a tool to help all open source engineers in the world build quick landing pages for their GitHub projects.

How does it work?

1. You create a gitpage.yml file and choose what blocks you want
2. You visit gitpages.app with the repo in the url
3. Your landing page is generated

The whole idea to allow people to be creative with their landing pages (vs just a README.md), but not everyone has the ability to create landing pages so I thought I would help.

The tool will be free, and is still in development, but you can get an understanding on the website and videos.

Would love to know what you think?

https://gitpages.app

- Dave

**Comment by jai-dewani** [comment]
Daam this looks impressive! I have signed up for the early access, hoping to use this on my next project or create one for my existing one 😌

**Reply by boyney123** [reply]
Thank you @jai-dewani!

**Comment by Agrover112** [comment]
Really loved this! Hope I can get early access to use it!

**Reply by boyney123** [reply]
Cool thanks @Agrover112 , yeah just sign up and I will update you over the next few weeks how to get started :)
